.travis.yml

```diff
@@ -4,9 +4,9 @@ dist: focal
 git:
   submodules: true
 before_install:
-  - repo_checks/submodule_check.sh
+  # - repo_checks/submodule_check.sh
+  - "true"
```

Comment:
Only disabled to show that it works with the not yet merged installer change. Hence the "DO NOT MERGE".